cluster/examples/kubernetes/ceph/object-openshift.yaml

```diff
@@ -101,7 +101,30 @@ spec:
     # Configure the pod liveness probe for the rgw daemon
     livenessProbe:
       disabled: false
-
+  # security oriented settings
+  # security:
+  # To enable the KMS configuration properly don't forget to uncomment the Secret at the end of the file
+  #   kms:
+  #     # name of the config map containing all the kms connection details
+  #     connectionDetails:
+  #        KMS_PROVIDER: "vault"
+  #        VAULT_ADDR: VAULT_ADDR_CHANGE_ME # e,g: https://vault.my-domain.com:8200
+  #        VAULT_BACKEND_PATH: "rook"
+  #        VAULT_SECRET_ENGINE: "kv"
+  #     # name of the secret containing the kms authentication token
+  #     tokenSecretName: rook-vault-token
+# UNCOMMENT THIS TO ENABLE A KMS CONNECTION
+# Also, do not forget to replace both:
+#   * ROOK_TOKEN_CHANGE_ME: with a base64 encoded value of the token to use
+#   * VAULT_ADDR_CHANGE_ME: with the Vault address
```

Comment:
Double comment so that a user can un-comment the whole block and not break the manifest.

```suggestion
# # UNCOMMENT THIS TO ENABLE A KMS CONNECTION
# # Also, do not forget to replace both:
# #   * ROOK_TOKEN_CHANGE_ME: with a base64 encoded value of the token to use
# #   * VAULT_ADDR_CHANGE_ME: with the Vault address
```